src/main/java/uk/ac/ebi/eva/commons/models/data/Variant.java

```diff
@@ -191,6 +191,7 @@ public final void setChromosome(String chromosome) {
         if (chromosome == null || chromosome.length() == 0) {
             throw new IllegalArgumentException("Chromosome must not be empty");
         }
+        this.chromosome = chromosome;
```

Comment:
Wow! I am wondering if this code path was ever invoked at all and how on earth it worked! Did you happen to perform any code search to see if there are any places where this setChromosome function was invoked?

Reply:
No could not find the usage of it.  In all the places the parameterized onstructor is being used